src/utils/index.ts

```diff
@@ -17,19 +17,44 @@ export function assertIsValidKeyGetter<R, K extends React.Key>(
     throw new Error('Please specify the rowKeyGetter prop to use selection');
   }
 }
-
-export function clampColumnWidth<R, SR>(
-  width: number,
+export function getColumnWidthForMeasurement<R, SR>(
```

Comment:
I believe that this function is not needed at all if we pass `min-width` and `max-width` to columns styles; then setting any `width` will not break the grid layout

Reply:
This is needed to fix this [bug](https://github.com/adazzle/react-data-grid/pull/3746#issuecomment-2744260592). We already set `min/max` width on the measuring cell